lib/middleware/request-handler.ts

```diff
@@ -494,3 +494,36 @@ export function onUnsubscribe(req: Interface.IOurRequest,
             return next(new restify.InternalError(err.message));
         });
 }
+
+export function onChangeSubscriptions(req: Interface.IOurRequest,
+                                      res: Interface.IOurResponse,
+                                      next: restify.Next): void
+{
+    if (!req.apiSession) {
+        req.log.error('No apisession object found.');
+        return next(new restify.InternalError('No apisession object found.'));
+    }
+
+    if (!req.blpSession) {
```

Comment:
Not sure, this was the existing code (moved to a new function) and may have just been defensive. @ericlu88 ?